apps/wallet/src/ui/app/shared/page-main-layout/PageMainLayout.tsx

```diff
@@ -107,13 +105,13 @@ function LeftContent({
         >
             <div
                 className={cn(
-                    'flex h-8 w-8 items-center justify-center rounded-full bg-primary-30 [&_svg]:h-5 [&_svg]:w-5 [&_svg]:text-white',
+                    'flex h-8 w-8 shrink-0 items-center justify-center rounded-full bg-primary-30 [&_svg]:h-5 [&_svg]:w-5 [&_svg]:text-white',
                     backgroundColor,
                 )}
             >
                 {isLedgerAccount ? <Ledger /> : <IotaLogoMark />}
             </div>
-            <span className="shrink-0 text-title-sm text-neutral-10 dark:text-neutral-92">
+            <span className="line-clamp-1 text-title-sm text-neutral-10 dark:text-neutral-92">
```

Comment:
the settings icon cannot be seen
![image](https://github.com/user-attachments/assets/935365e1-66ef-49c3-9227-a7f1ff846539)

Reply:
did you rebuild it? because it works for me :see_no_evil: 
![image](https://github.com/user-attachments/assets/d707b1fa-0e37-41a8-8ed2-9385244da534)
![image](https://github.com/user-attachments/assets/59ae2743-f3d5-4b7e-b821-5a9fb75c4ac4)